catalog/plugin/catalog-plugin-metacard-validation/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -48,15 +47,13 @@
     <!-- Pre-Query Metacard Validity Checker Plugin -->
     <bean id="pre-query-plugin"
           class="ddf.catalog.metacard.validation.MetacardValidityCheckerPlugin">
-        <!--<cm:managed-properties-->
-        <!--persistent-id="MetacardValidityCheckerPlugin"-->
-        <!--update-strategy="container-managed"/>-->
+        <cm:managed-properties
```

Comment:
You don't need to declare the property in blueprint. Really, you should only be initializing it in one place. Either initialize the variable in the class or initialize it via blueprint with a property, but not both.